code-challenges.test.js

```diff
@@ -28,42 +28,161 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+// const num1 = 15
+// // Expected output: "15 is divisible by three"
+// const num2 = 0
+// // Expected output: "0 is divisible by three"
+// const num3 = -7
+// // Expected output: "-7 is not divisible by three"
+
+//a describe method that lists the name of a function called divisibleByThree test.
+describe("divisibleByThree", () => {
+    // a test/it method, nested within the describe block, that in plain words, describes what that the function does.
+    it("returns a string that tells you if the number is divisible by three", () => {
+        const num1 = 15
 // Expected output: "15 is divisible by three"
-const num2 = 0
+        const num2 = 0
 // Expected output: "0 is divisible by three"
-const num3 = -7
+        const num3 = -7
 // Expected output: "-7 is not divisible by three"
+        //an expect method, nested within the test block, calling on the divisibleByThree function, followed by the .toEqual() matcher that checks the expected output of the function return.
+        expect(divisibleByThree(15)).toEqual("is divisible by three")
+        expect(divisibleByThree(0)).toEqual("is divisible by three")
+        expect(divisibleByThree(-7)).toEqual("is not divisible by three")
+    })
+})
```

Comment:
ooo, false positive testing. Notice how the Expected output says something like 
`"0 is divisible by three"`
but your test only looks for 
`"is divisible by three"`

That means although your test will pass you are not achieving what was desired by your stakeholders, be they clients or a company. 

Good work either way! Great code just keep an eye out for these kind of situations!